plugins/SitesManager/vue/src/SitesManagement/SitesManagement.vue

```diff
@@ -67,21 +67,29 @@
       <div class="ui-confirm add-site-dialog">
         <div>
           <h2>{{ translate('SitesManager_ChooseMeasurableTypeHeadline') }}</h2>
-
           <div class="center">
-            <p>
+            <p>{{ translate('SitesManager_ChooseMeasurableTypeSubheader') }}</p>
+            <br>
+          </div>
+          <div class="card-row">
+            <ContentBlock
+              v-for="type in availableTypes"
+              :key="type.id"
+              :content-title="type.name"
+            >
+              <p class="center">
+                {{ type.longDescription }}
+              </p>
               <button
                 type="button"
-                v-for="type in availableTypes"
-                :key="type.id"
                 :title="type.description"
-                class="modal-close btn"
-                @click="addSite(type.id);"
+                class="modal-close btn btn-block"
```

Comment:
Seems like the full width from the `btn-block` class is not needed here, it already has it, but at the same time, I guess, it's clearer to be explicit, so all good to keep it.